src/UKHO.ADDS.EFS.Domain/RetryPolicy/HttpClientPolicyProvider.cs

```diff
@@ -0,0 +1,172 @@
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using Polly;
+using UKHO.ADDS.Infrastructure.Results;
+
+namespace UKHO.ADDS.EFS.Domain.RetryPolicy
```

Comment:
for domain project the namespace must not include Domain.
<RootNamespace>UKHO.ADDS.EFS</RootNamespace> defined csproj file.

```suggestion
namespace UKHO.ADDS.EFS.RetryPolicy
```